Client.xcodeproj/xcshareddata/xcschemes/Fennec.xcscheme

```diff
@@ -140,8 +140,8 @@
    </BuildAction>
    <TestAction
       buildConfiguration = "Fennec"
-      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
-      selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      selectedDebuggerIdentifier = ""
+      selectedLauncherIdentifier = "Xcode.IDEFoundation.Launcher.PosixSpawn"
```

Comment:
are these changes coming after the hack session with @clarmso ? I'm not sure about those, would prefer a confirmation for her who worked with Orla on this to prevent the simulator black screen issue